Applications/TransferLearning/Draw_Classification/jni/meson.build

```diff
@@ -15,7 +18,7 @@ training_sources = [
 
 e = executable('nntrainer_training',
   training_sources,
-  dependencies: [app_utils_dep, iniparser_dep, nntrainer_capi_dep, nnstreamer_capi_dep, tflite_dep],
+  dependencies: [app_utils_dep, iniparser_dep, nntrainer_capi_dep, nnstreamer_capi_dep, tflite_dep, gtest_dep],
```

Comment:
I think this should be separated according to get_option("enable-test")

Reply:
then how about we add -DENABLE_TEST as well. So that applications will only run test with `enable-test`?

Reply:
> then how about we add -DENABLE_TEST as well. So that applications will only run test with `enable-test`?

Sounds good to me.!